protocols/v2/subprotocols/mining/src/new_mining_job.rs

```diff
@@ -51,59 +64,70 @@ impl<'d> NewMiningJob<'d> {
     }
 }
 
-/// NewExtendedMiningJob (Server -> Client)
+/// Message used by an upstream to provide an updated mining job to the downstream through
+/// Extended or Group Channel only.
+///
+/// An Extended Job allows rolling Merkle Roots, giving extensive control over the search space
+/// so that they can implement various advanced use cases such as: translation between Sv1 and Sv2
+/// protocols, difficulty aggregation and search space splitting.
+///
+/// In the Extended Channel scenario, the full coinbase transaction can be constructed as of the
+/// following: `extranonce_prefix + extranonce`, which should equal the negotiated extranonce space
+/// for the channel [`extranonce_size`].
+///
+/// In the Group Channel scenario, the full coinbase can then be constructed as the following:
+/// [`NewExtendedMiningJob::coinbase_tx_prefix`] + `extranonce_prefix` + `extranonce` +
+/// [`NewExtendedMiningJob::coinbase_tx_suffix`].
 ///
-/// (Extended and group channels only)
-/// For an *extended channel*: The whole search space of the job is owned by the specified
-/// channel. If the future_job field is set to *False,* the client MUST start to mine on the new job
-/// as soon as possible after receiving this message.
-/// For a *group channel*: This is a broadcast variant of NewMiningJob message with the
-/// merkle_root field replaced by merkle_path and coinbase TX prefix and suffix, for further traffic
-/// optimization. The Merkle root is then defined deterministically for each channel by the
-/// common merkle_path and unique extranonce_prefix serialized into the coinbase. The full
-/// coinbase is then constructed as follows: *coinbase_tx_prefix* + *extranonce_prefix* +
-/// *coinbase_tx_suffix*. The proxy MAY transform this multicast variant for downstream standard
-/// channels into NewMiningJob messages by computing the derived Merkle root for them. A proxy MUST
-/// translate the message for all downstream channels belonging to the group which don’t signal
-/// that they accept extended mining jobs in the SetupConnection message (intended and
-/// expected behaviour for end mining devices).
+/// [`extranonce_size`]: crate::OpenExtendedMiningChannelSuccess::extranonce_size
```

Comment:
seem like we're leaving this line behind... is that intentional?

Reply:
this is a reference to the variables, so in the docs the user can click on extranonce_size and be redirected to the correct place